src/textual/screen.py

```diff
@@ -169,6 +176,18 @@ def _update_timer(self) -> Timer:
             )
         return self.__update_timer
 
+    @property
+    def layers(self) -> tuple[str, ...]:
+        """Layers of from parent.
```

Comment:
Not sure what you meant here?

```suggestion
        """Layers of names from parent.
```

```suggestion
        """Layers from parent.
```